src/BTCPayServer.Lightning.LNbank/LNbankClient.cs

```diff
@@ -42,6 +42,18 @@ public async Task<InvoiceData> GetInvoice(string invoiceId, CancellationToken ca
             return await Get<InvoiceData>($"invoice/{invoiceId}", cancellation);
         }
 
+        public async Task<InvoiceData[]> ListInvoices(ListInvoicesParams param, CancellationToken cancellation)
+        {
+            var path = new StringBuilder("invoices");
+            if (param != null)
+            {
+                if (param is { PendingOnly: true }) path.Append("pending_only=true&");
```

Comment:
there is a bug here, if `param.OffsetIndex` isn't specified, the path will end with a `&` which is invalid query string.

Reply:
Thanks, updated in [1932743](https://github.com/btcpayserver/BTCPayServer.Lightning/pull/99/commits/19327433178e3f922bd92cf3ec67409767aab872)!